pocs/mount/mount.py

```diff
@@ -72,8 +72,8 @@ def __init__(self, location, commands=None, *args, **kwargs
         self._state = 'Parked'
 
         self.sidereal_rate = ((360 * u.degree).to(u.arcsec) / (86164 * u.second))
-        self.ra_guide_rate = 0.5  # Sidereal
-        self.dec_guide_rate = 0.5  # Sidereal
+        self.ra_guide_rate = 0.9  # Sidereal
```

Comment:
Does guide rate mean the speed with which we make adjustments to the position between images?

Reply:
This is documented somewhere but the fact that I don't know where and that you have a question here makes that irrelevant.

I want to turn these into properties of the class, which would be more consistent with all these other values and would allow for an easier docstring. - #552